src/Util.cc

```diff
@@ -321,21 +321,41 @@ ignition::common::SystemPaths *ignition::common::systemPaths()
 
 /////////////////////////////////////////////////
 bool ignition::common::env(const std::string &_name, std::string &_value)
+{
+  return env(_name, _value, false);
+}
+
+/////////////////////////////////////////////////
+bool ignition::common::env(const std::string &_name, std::string &_value,
+                           bool _allowEmpty)
 {
   std::string v;
+  bool valid = false;
 #ifdef _WIN32
   const DWORD buffSize = 32767;
   static char buffer[buffSize];
   if (GetEnvironmentVariable(_name.c_str(), buffer, buffSize))
   {
     v = buffer;
   }
+
+  if(!v.empty())
+    valid = true;
+
```

Comment:
I'm still thinking about how this may affect Windows unintentionally. How about printing a warning if `_allowEmpty == true` on Windows?